Utility/treeHelper.FCMacro

```diff
@@ -0,0 +1,338 @@
+# coding: utf-8
+
+__Name__ = "Tree Helper"
+__Comment__ = "Macro to generate a helper tree that allows you to easily move objects in the tree structure"
+__Author__ = "Daniel Pose"
+__Version__ = "0.0.4"
+__Date__ = "2022-09-04"
+__License__ = "MIT"
+__Web__ = "https://github.com/iinjdpa/TreeHelper"
+__Wiki__ = ""
+__Icon__ = "/treeHelper/treeHelper.svg"
+__Help__ = "Select a folder, part or body where helper tree must grow. Drag object from main tree to helper tree"
+__Status__ = "Alpha"
+__Requires__ = "Freecad >= v0.19"
+__Communication__ = "https://github.com/iinjdpa/TreeHelper/issues"
+__Files__ = "/treeHelper/ui_treewindow.py,/treeHelper/treeHelper.svg"
+
+
+import FreeCAD as App
+import FreeCADGui as Gui
+from PySide import QtCore, QtGui
+from PySide.QtCore import Qt, QByteArray, QDataStream, QIODevice
+from PySide.QtGui import QDialog, QApplication, QTreeWidgetItem, QDockWidget, QGraphicsDropShadowEffect, QColor,QMainWindow
+from PySide.QtGui import QMessageBox, QAbstractItemView, QDrag
+import PySide2
+import os
+import sys
+from treeHelper import ui_treewindow
+from collections import OrderedDict
+import time
+
+
+class Objdoc(object):
+    def __init__(self,doc):
+        self.ViewObject = mViewObject()
+        self.Name = doc.Name
+        self.Label = doc.Name
+
+class mViewObject(object):
+    def __init__(self):
+        pass
+    def claimChildren(self):
+        # Nombre del documento
+        doc = App.activeDocument()
+        if doc == None:
+            doc = FreeCAD.newDocument()

```

Comment:
Why do you create a new document? I though that the macro helps to sort the tree. A new document has no objects to sort.